packages/p2p/src/components/buy-sell/row.jsx

```diff
@@ -29,21 +29,30 @@ BuySellRowLoader.propTypes = {
 export const RowComponent = React.memo(({ data, is_buy, setSelectedAd, style }) => {
     const { advertiser_id } = React.useContext(Dp2pContext);
     const is_own_ad = data.advertiser_id === advertiser_id;
+    const short_name = data.advertiser_name.substr(0, 2).toUpperCase();
+    const colors = ['#f43f83', '#85acb0', '#9ed178', '#ff6444'];
+    const random_color = colors[Math.floor(Math.random() * colors.length)];
```

Comment:
why are we randomizing the background colour? can you add a comment in the code to explain that?

Reply:
sure. This background color is for the use of user icon. It's completely random.
<img width="642" alt="Screenshot 2020-04-17 at 10 42 03 AM" src="https://user-images.githubusercontent.com/51310435/79526574-26a5b600-8098-11ea-9674-8c5ddc33a2da.png">